tests/e2e/e2e_metrics_test.go

```diff
@@ -28,7 +28,11 @@ var _ = OSMDescribe("Custom WASM metrics between one client pod and one server",
 			// Install OSM
 			installOpts := Td.GetOSMInstallOpts()
 			installOpts.DeployPrometheus = true
-			installOpts.SetOverrides = []string{"OpenServiceMesh.enableWASMStatsExperimental=true"}
+			installOpts.SetOverrides = []string{
+				"OpenServiceMesh.enableWASMStatsExperimental=true",
```

Comment:
Could you add a comment on why these values.

Reply:
they are just the previous ones we had, where things used to work. I don't know the exact values accepted by CI tbh

Reply:
Even that in itself can be documented.